cmake/config.h.in

```diff
@@ -0,0 +1,43 @@
+#cmakedefine HAVE_FORK 1
+#cmakedefine HAVE_LSEEK64 1
+#cmakedefine HAVE_LUA 1
+#cmakedefine HAVE_POSIX_FADVISE 1
+#cmakedefine HAVE_POSIX_FALLOCATE 1
+#cmakedefine HAVE_PTHREAD 1
+#cmakedefine HAVE_SYNC_FILE_RANGE 1
+#cmakedefine HAVE_TERMIOS_H 1
+#cmakedefine HAVE_LIBGEN_H 1
+#cmakedefine HAVE_SYS_WAIT_H 1
+#cmakedefine HAVE_UNISTD_H 1
+#cmakedefine SIZEOF_OFF_T ${SIZEOF_OFF_T}
+
+#ifdef _MSC_VER
+#if _MSC_VER < 1900
+#define snprintf _snprintf
+#endif
+
+typedef __int64 ssize_t;
```

Comment:
This is wrong for 32 bit Windows. No idea if anyone still cares. But if you do, there seems to be a correct define on Windows which you can use: https://msdn.microsoft.com/de-de/library/windows/desktop/aa383751(v=vs.85).aspx